### DISCUSSION TITLE
Make reverting changes in skin editor a dangerous action

### DISCUSSION THREAD

**Discussion by mk56-spn** [ppy/osu#22740] [Ideas] [answered]
https://user-images.githubusercontent.com/74463310/221402505-18b1d3f6-5272-4e28-8279-4122fd914366.mov

Just seems like a precaution that should be had as the complexity of lazer skins grow

**Comment by Axilotl17** [comment]
I think "This will reset the skin to the state it was in when first imported" is more digestible

**Reply by mk56-spn** [reply]
yeah agreed, this was just a message i came up with in half a min to demo this

**Comment by mk56-spn** [comment, marked as answer]
will be closed by https://github.com/ppy/osu/pull/22747
